modules/rostests/win32/user32/serverside/serverside.c

```diff
@@ -0,0 +1,107 @@
+/*
+ * PROJECT:         ReactOS test suite
+ * LICENSE:         GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:         Test for existense of the kernel mode wndproc in the window
+ * COPYRIGHT:       Copyright 2024 Oleg Dubinskiy (oleg.dubinskiy@reactos.org)
+ */
+
+/* This testapp tests behaviour of IsServerSideWindow() function in user32. */
+
+#define _WINE
+
+#include <windows.h>
+#include <stdio.h>
+
+#include <debug.h>
+
+WCHAR WndClass[] = L"window class";
+
+LRESULT CALLBACK WndProc(HWND hWnd,
+                         UINT msg,
+                         WPARAM wParam,
+                         LPARAM lParam)
+{
+    switch (msg)
+    {
+        case WM_PAINT:
+            return 0;
+
+        case WM_DESTROY:
+            PostQuitMessage(0);
+            return 0;
+    }
+
+    return DefWindowProcW(hWnd, msg, wParam, lParam);
+}
+
+int APIENTRY wWinMain(HINSTANCE hInst,
+                      HINSTANCE hPrevInstance,
+                      LPWSTR lpCmdLine,
+                      int nCmdShow)
+{
+    HWND hWnd1a, hWnd1b;
+    WNDCLASSEXW wcx;
+    UINT result;
+    BOOL ret;
+
+    memset(&wcx, 0, sizeof(wcx));
+    wcx.cbSize = sizeof(wcx);
+    wcx.lpfnWndProc = (WNDPROC)WndProc;
+    wcx.hInstance = hInst;
+    wcx.hbrBackground = (HBRUSH)COLOR_WINDOW;
```

Comment:
`(HBRUSH)(COLOR_WINDOW + 1)`?

Reply:
Does this increment set another background colour for the window? Or it has some another purpose to change this? :thinking: 
It's just based on user32_sysicon apitest, so I did not change some parts of the previous code so much yet.

Reply:
https://devblogs.microsoft.com/oldnewthing/20140305-00/?p=1593

Reply:
OK, then it makes sense.
Applied the suggestion. Thanks.